### ISSUE TITLE
"Generate all project documentation" missing?

### ISSUE THREAD

**Issue by elpollodiablo84** [TerosTechnology/vscode-terosHDL#458]
Maybe I'm wrong, but it seems to me that after the big v5 update the "Generate all project documentation"(*) feature is missing. If I'm not wrong, I would suggest to reintroduce the feature since it was really helpful.

(*): Just to be clear, I'm referring to the possibility to create a folder with the documentation of all the modules of the project, with hyperlinks between files and a main page with the project description.

**Comment by qarlosalberto** [comment]
Yes, it's missing. I plan to do a refactor and I will add it in the future.

**Comment by polyee13** [comment]
I'm also interested in this. Is it an alternative to the cli version? https://github.com/TerosTechnology/teroshdl-documenter-demo

**Comment by qarlosalberto** [comment]
For now there isn't an alternative.
